src/docs/stan-reference/distributions.tex

```diff
@@ -1117,7 +1117,7 @@ \subsubsection{Stan Functions}
 \end{description}
 %
 \begin{description}
-\fitem{real}{normal\_rng}{real \farg{mu},  real \farg{sigma}}{Generate a normal
+\fitem{real}{normal\_rng}{reals \farg{mu},  reals \farg{sigma}}{Generate a normal
```

Comment:
The return type and behavior under vectorization needs to be specified.  It needs to answer at least these questions:

* What is return type if all arguments are `real`?
* Can one argument be `real` and the other `vector`?  If so, what's the return type?
* Same question for `vector` and `row_vector` and for `vector` and `real[]` (array of real)?

The answers should be written up in a section on vectorized RNGs.  It can go with the rest of the probability function vectorization descriptions.  But it should be in its own section.  Then these definitions should provide a short link, as in "see \refsec{vec-sec-name} for argument and return type details."